README.md

```diff
@@ -9,43 +9,74 @@ The process for submitting an RFC is documented in the [RFC Process](CONTRIBUTIN
 
 For the main reference implementation of the ILP stack, see [Interledger.js](https://github.com/interledgerjs).
 
-## Index
-
-Here is a list of the main Interledger RFCs:
+## Interledger Overview and Explanatory Docs
 
 * **[1: Interledger Architecture](0001-interledger-architecture/0001-interledger-architecture.md)**
 
   Overview of the Interledger architecture.
 
-* **[3: Interledger Protocol (ILP)](0003-interledger-protocol/0003-interledger-protocol.md)**
+* **[19: Glossary](./0019-glossary/0019-glossary.md)**
 
-  Specifies the Interledger Protocol and Interledger Packet, which are used for sending payment instructions across different ledgers and connectors. This is the core protocol in the Interledger stack.
+  Definitions of Interledger terminology.
 
-* **[4: Ledger Plugin Interface](0004-ledger-plugin-interface/0004-ledger-plugin-interface.md)**
+* **[20: Explain Like I'm Five (ELI5)](0020-explain-like-im-five/0020-explain-like-im-five.md)**
 
-  JavaScript interface for ledger plugins, which implement all functionality necessary to make Interledger payments through a ledger. This can be used as a model for defining such plugins in other languages.
+  Basic explanation of Interledger.
 
-* **[8: Interledger Quoting Protocol (ILQP)](0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md)**
+* **[22: Hashed-Timelock Agreements (HTLAs)](0022-hashed-timelock-agreements/0022-hashed-timelock-agreements.md)**
 
-  Specifies the protocol used for requesting a quote for an Interledger payment from a connector.
+  How conditional transfers can be implemented over any type of ledger, including those that do not natively support Hashed-Timelock Contracts.
 
-* **[9: Simple Payment Setup Protocol (SPSP)](0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md)**
+## Core Interledger Protocol Specs
 
-  A basic Application Layer protocol that uses HTTPS to exchange details needed to set up an Interledger payment.
 
-* **[10: Connector-To-Connector Protocol (CCP)](0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md)**
+* **[3: Interledger Protocol (ILP)](0003-interledger-protocol/0003-interledger-protocol.md)**
 
-  The protocol used by connectors to communicate routing information.
+  Specifies the Interledger Protocol and Interledger Packet, which are used for sending payment instructions across different ledgers and connectors. This is the core protocol in the Interledger stack.
 
-* **[11: Interledger Payment Request (IPR)](0011-interledger-payment-request/0011-interledger-payment-request.md)**
+* **[8: Interledger Quoting Protocol (ILQP)](0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md)**
 
-  A Transport Layer protocol in which the receiver generates the ILP Packet and condition.
+  Specifies the protocol used for requesting a quote for an Interledger payment from a connector.
```

Comment:
maybe "The protocol for quoting an interledger payment, by sending a request to the connector"